### PR TITLE
Adds HTML attributes for iOS web views

### PR DESCRIPTION
**Pull request by ashfurrow**
Just noticed the TypeScript errors, will pick it up from there tomorrow.

### CODE REVIEW

**Comment by ashfurrow** [comment]
Looks like React has already added attributes to take care of this, but are camel-cased, so I just need to use those instead.

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit f0a4f0eac9d4499671f3b2924e1bd868febb3195

https://deploy-preview-1297--artsy-reaction.netlify.com

**Comment by ashfurrow** [comment]
We should go through and QA these with Brian. I've made decisions based on what I think would be the best iOS UX, but it's always better to have a second opinion.

File: src/Apps/Order/Components/AddressForm.tsx [hunk 1]
```diff
@@ -77,6 +77,8 @@ export class AddressForm extends React.Component<
             id="AddressForm_name"
             placeholder="Add full name"
             title="Full name"
+            autoCapitalize="words"
+            autoCorrect="off"
```
Comment (sweir27) [inline src/Apps/Order/Components/AddressForm.tsx]: 🙌

**Comment by sweir27** [comment]
This looks pretty straightforward! Agreed on syncing with @briansw on this so we don't have to come back later, but other than that is good with me. 😄

**Comment by ashfurrow** [comment]
Okay, I'm going to merge since getting this onto staging is the easiest way for Brian to see it. I'll take care of following up.

**Pull request merged**